src/metabase/api/pulse.clj

```diff
@@ -126,12 +130,14 @@
 
 (defendpoint POST "/test"
   "Test send an unsaved pulse."
-  [:as {{:keys [name cards channels] :as body} :body}]
+  [:as {{:keys [name cards channels skip] :as body} :body}]
   {name     su/NonBlankString
    cards    (su/non-empty [su/Map])
-   channels (su/non-empty [su/Map])}
+   channels (su/non-empty [su/Map])
+   skip     s/Bool
+   }
   (check-card-read-permissions cards)
   (p/send-pulse! body)
-  {:ok true})
+  (merge body {:ok true}))
```

Comment:
Fixing as you suggest.

This change helped me for debugging purposes, should I leave it unchanged and propose this in a separate issue?